tests/ai_reports/factories.py

```diff
@@ -10,8 +10,8 @@ class AiReportFactory(factory.django.DjangoModelFactory):
     class Meta:
         model = AiReport
 
-    category = factory.Faker("word")
-    explanation = factory.Faker("sentence", nb_words=6)
+    label = factory.Faker("word")
```

Comment:
The label and the confidence coming from XAI API are also a list. Why not use a list here too?